content/en/data_jobs/databricks.md

```diff
@@ -137,6 +139,39 @@ Optionally, you can also set other init script parameters and Datadog environmen
 
 {{< /tabs >}}
 
+### Advanced settings (optional)
+
+1. set `DD_RUN_NAME` environment variable using `spark_env_vars` field for your one-time job run.
```

Comment:
This needs a clear header for "Why?" you are doing this in the first place before we go into the specifics of the change. Something like `Aggregate runs for the same job when using orchestration outside of Databricks (e.g. Airflow, Azure Data Factory)`

Reply:
I agree that calling out the utility as a first line or header will help customers know when this will be relevant.

But, we haven't looked closely at how well this change satisfies issues arising from orchestration platforms like ADF and Airflow, so it seems prudent to omit citing them specifically. We can avoid any missteps that might arise from not testing the updates against those contexts.

Maybe we can keep things rooted in the Databricks API for now? 

>"Aggregate cluster metrics from one-time runs (ie [`/jobs/runs/submit`](https://docs.databricks.com/api/workspace/jobs/submit)) by the `run_name`. 
>
>One time job runs created with the ... API are given unique job IDs, which results in each run having a row in the clusters table. These runs can be aggregated on ... (etc)

WDYT?

Reply:
updated the header following your suggestions, left the details out (e.x submit api path and the detailed field name) with only a high level utility.